src/components/ConnectToQuickbooksOnlineButton/index.tsx

```diff
@@ -40,6 +40,8 @@ function ConnectToQuickbooksOnlineButton({policyID, shouldDisconnectIntegrationB
             {shouldDisconnectIntegrationBeforeConnecting && integrationToDisconnect && isDisconnectModalOpen && (
                 <AccountingConnectionConfirmationModal
                     onConfirm={() => {
+                        // Since QBO doesn't support Taxes, we should disable them from the LHN when connecting to QBO
+                        PolicyAction.enablePolicyTaxes(policyID, false);
```

Comment:
NAB: The name of this command is misleading IMO and it should be renamed to `togglePolicyTaxes` since you pass `false` to it to disable policy taxes.